File: C/Assignments/Assignment 1/q4.c
// 4. Write a program to check whether a given character is a vowel or consonant.

void main() {
	
	char ch;

    printf("Enter character: ");
    scanf("%c", &ch);

    if((ch >= 'a' && ch <= 'z') || (ch >= 'A' && ch <= 'Z')) {

        if(ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u' ||
           ch == 'A' || ch == 'E' || ch == 'I' || ch == 'O' || ch == 'U') {

            printf("%c is a vowel", ch);
        }
        else {
            printf("%c is a consonant", ch);
        }
    }
    else {
        printf("%c is not an alphabet", ch);
    }
    
}

File: C to Java/Assignments/Assignment 1/q4.java
// 4. Write a program to check whether a given character is a vowel or consonant.

import java.util.Scanner;

public class q4 {

    public static void main(String args[]) {

        Scanner sc = new Scanner(System.in);

        //take input
	char ch ;
	System.out.printf("enter charchter : ");
	ch = sc.next().charAt(0);
	
//	check and print given character is a vowel or consonant
	if(ch =='a' || ch =='e' || ch =='i' || ch =='o' || ch =='u' || ch =='A' || ch =='E' || ch =='I' || ch =='O' || ch =='U' ) {
		System.out.printf("%c is a vowel",ch);
	}
	
	else {
		System.out.printf("%c is consonant", ch);
	}

        }
}